package.json

```diff
@@ -4,22 +4,22 @@
   "private": true,
   "dependencies": {
     "@shopify/polaris": "^4.19.0",
+    "@shopify/polaris-icons": "^4.5.0",
     "@testing-library/jest-dom": "^4.2.4",
     "@testing-library/react": "^9.3.2",
     "@testing-library/user-event": "^7.1.2",
     "react": "^16.13.1",
     "react-dom": "^16.13.1",
-    "react-scripts": "3.4.1",
+    "react-scripts": "4.0.3",
```

Comment:
It appears the codebase was possible produced from an older js-oriented codebase? I upgraded to the latest version of CRA. `testing-library` should be updated as well, but i didn't want to touch even more in this ticket.